docs/modules/attacks.rst

```diff
@@ -14,7 +14,7 @@ Base Class Evasion Attacks
 
 Base Class Poisoning Attacks
 ----------------------------
-.. autoclass:: PoisoningAttack
+.. autoclass:: PoisoningAttackBlackBox
```

Comment:
I think we also need to add PoisoningAttackWhiteBox here.

Reply:
Will do. Fixed in commit.